### ISSUE TITLE
PNG To JPG Converter

### ISSUE THREAD

**Issue by neelshah2409** [prathimacode-hub/Awesome_Python_Scripts#872]
Title: PNG To JPG Converter
About: It's an GUI script
Name: Neel Shah
Label: Feature Request
Assignee: ''

Define You:

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant
 - [ ] Contributor

Here using the tkinter we will make gui where there will be two buttons one for selecting the png file and other for converting the file. We will first select the png image and then we will give the name for saving the jpg imag

**Comment by neelshah2409** [comment]
Please see this issue and assign me and had one query where to put the script in the gui section or image processing section?

**Comment by prathimacode-hub** [comment]
This is already done I guess. @neelshah2409

**Comment by neelshah2409** [comment]
> This is already done I guess. @neelshah2409

Mam you can check it once ,please. If it is done than I would go with other one.

**Comment by neelshah2409** [comment]
> This is already done I guess. @neelshah2409

@prathimacode-hub Mam any update whether it is done or not?

**Comment by prathimacode-hub** [comment]
You can check it [here](https://github.com/prathimacode-hub/Awesome_Python_Scripts/tree/main/BasicPythonScripts/Document%20Converter%20App). You can do this as an enhancement if you would like to. Let me know of your interest so that I can assign. @neelshah2409
